packages/web-api-scan-job-manager/src/batch/batch.ts

```diff
@@ -142,11 +147,18 @@ export class Batch {
         const jobTasks: Map<string, JobTask> = new Map();
         const taskAddParameters: BatchServiceModels.TaskAddParameter[] = [];
         const maxTaskDurationInMinutes = (await this.getTaskConfig()).taskTimeoutInMinutes;
+        let sasUrl: string;
+
+        try {
+            sasUrl = await this.containerSASUrlProvider.generateSASUrl(Batch.batchLogContainerName);
+        } catch (error) {
```

Comment:
Should we just fail if unable to create SAS? What anticipated condition expected when SAS is not available? We do normally expect to create SAS, correct?